Dockerfile

```diff
@@ -19,8 +35,10 @@ RUN addgroup -S ${XP_USER_GROUP} \
     && mkdir /app \
     && chown -R ${XP_USER}:${XP_USER_GROUP} /app
 
-COPY --chown=${XP_USER}:${XP_USER_GROUP} management-service/bin/* /app/
-COPY --chown=${XP_USER}:${XP_USER_GROUP} management-service/database /app/database/
+COPY --chown=${XP_USER}:${XP_USER_GROUP} management-service/bin/experiments.yaml /app/
```

Comment:
Nit: Can we do `COPY --chown=${XP_USER}:${XP_USER_GROUP} management-service/bin/*.yaml /app/` instead of 2 separate commands to copy the OpenAPI specs?